src/test/templates/sam/samTemplateGenerator.test.ts

```diff
@@ -122,70 +122,54 @@ describe('SamTemplateGenerator', function () {
     })
 
     it('errs if resource name is missing', async function () {
-        const error: Error = await assertThrowsError(async () => {
-            await new SamTemplateGenerator()
+        assert.rejects(
```

Comment:
```suggestion
        await assert.rejects(
```
You can also return the promise

Reply:
🤦